docs/src/pages/components/cdk-components/applicationLayout/api.jsx

```diff
@@ -15,16 +15,29 @@ const SidenavApplicationLayoutPropsTable = () => {
           <code>'overlay'</code>
         </td>
         <td>
-          Default action over the content of the page, overlay the content or
-          push to the right.
+          Default action over the content of the page, overlay the content or push to the right
+          ('push' | 'overlay'). In lower resolutions the mode will always be overlay.
         </td>
       </tr>
       <tr>
         <td>displayArrow: boolean</td>
         <td>
           <code>true</code>
         </td>
-        <td>If false, the arrow button is hidden.</td>
+        <td>
+          If false, the arrow button is hidden. In lower resolutions the arrow will be always
+          displayed.
+        </td>
+      </tr>
+      <tr>
+        <td>padding: string | object </td>
+        <td></td>
+        <td>
+          Size of the padding to be applied to the custom area ('xxsmall' |
+          'xsmall' | 'small' | 'medium' | 'large' | 'xlarge' | 'xxlarge'). You
+          can pass an object with 'top', 'bottom', 'left' and 'right' properties
+          in order to specify different padding sizes.
+        </td>
```

Comment:
This prop is from DxcSidenav. Maybe is not necessary to indicate it since we link to DxcSidenav props.

Reply:
What about this @Jialecl ?